cloud/shared/bin/destroy.py

```diff
@@ -0,0 +1,14 @@
+#! /usr/bin/env python3
+
+from cloud.shared.bin.lib import terraform
+from cloud.shared.bin.lib.setup_class_loader import get_config_specific_destroy
+"""
+Destroy.py destroys Civiform deployment.
+"""
```

Comment:
Module description should be at the beginning of the file
https://google.github.io/styleguide/pyguide.html#382-modules